courses/views_tests.py

```diff
@@ -1,12 +1,12 @@
 """Tests for the API"""
 # pylint: disable=no-self-use
 from django.core.urlresolvers import reverse
-from django.test import TestCase
 
 from .factories import ProgramFactory, CourseFactory
+from search.base import ESTestCase
 
 
-class ProgramTests(TestCase):
+class ProgramTests(ESTestCase):
```

Comment:
We need to run `recreate_index()` before each test to allow signals sent when the models change to connect to Elasticsearch correctly